tutorials/tour/anonymous-function-syntax.md

```diff
@@ -51,3 +51,33 @@ Function1[Int, Int]
 Function2[Int, Int, String]
 Function0[String]
 ```
+
+The following example shows how to use anonymous function of the beginning of this page
+
+```tut
+package tour
```

Comment:
The `tut` error you're getting will go away if you simply omit this line, which isn't necessary anyway.